content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -120,6 +118,50 @@ The release captain has unlimited power to make changes to the release branch to
 
 Most issues are non-blocking. Fixes to non-blocking issues can be fixed in `main` by the code owner who can then `git cherry-pick` those commits into the release branch with the approval of the release captain. Alternatively, broken features can be reverted out of the release branch or disabled via feature flags if they aren't ready or are too buggy.
 
+## Minor release process
+
+### 1) Start a minor release
+
+Major and minor releases are released on a fixed scheule, see [when we release](#when-we-release).
+
+### 2) Minor release tracking issue
+
+The tracking issue for the current minor release is created as a part of the post-release step from the previous minor release. [3.36.0 release tracking issue] is a good example of a minor release tracking issue.
```

Comment:
Similar: We should just link to the template here, instead of a potentially outdated example